### PR TITLE
responses to assessment

### PR DESCRIPTION
**Pull request by baz1389**

### CODE REVIEW

**Comment by baz1389** [comment]
Line 33 should be hair_color, not eye_color. I read the question too fast.

File: assessment.sql [hunk 1]
```diff
@@ -5,24 +5,42 @@ CREATE DATABASE "sql-crud-assessment";
...
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  gender CHARACTER VARYING(1),
+  height INTEGER,
+  weight INTEGER,
+  dob DATE,
+  eye_color CHARACTER VARYING,
+  hair_color CHARACTER VARYING
+);
```
Comment (srhoulam) [inline assessment.sql:18]: Good. 1

File: assessment.sql [hunk 2]
```diff
@@ -5,24 +5,42 @@ CREATE DATABASE "sql-crud-assessment";
...
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
--- your code here
+INSERT INTO licensees(surname, given_name, gender, height, weight, dob, eye_color, hair_color)
+VALUES('Bishop', 'Caren', 'f', 63, 132, '1943-09-26', 'Brown', 'Black');
```
Comment (srhoulam) [inline assessment.sql:23]: Correct. 1

File: assessment.sql [hunk 3]
```diff
@@ -5,24 +5,42 @@ CREATE DATABASE "sql-crud-assessment";
...
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
--- your code here
+\copy licensees(surname, given_name, gender, height, weight, dob, eye_color, hair_color) FROM 'data/licensees.csv' WITH (HEADER true, FORMAT csv) --you don't need a semi-colon here, right?
```
Comment (srhoulam) [inline assessment.sql:27]: Yes, no semicolon is needed. This is a Postgres command, not a SQL statement. 1

File: assessment.sql [hunk 4]
```diff
@@ -5,24 +5,42 @@ CREATE DATABASE "sql-crud-assessment";
...
 
 -- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
 
--- your code here
+SELECT eye_color FROM licensees
+WHERE eye_color LIKE 'Hazel'
+  AND ((eye_color LIKE 'Brown') OR (eye_color LIKE 'Black')
+;
```
Comment (srhoulam) [inline assessment.sql:34]: ~~Excellent. 1~~

Reading your comment about line 33, I'll pretend it says `hair_color` instead. This WHERE clause is good, it just needs a closing paren at the end. Remember, you opened a set of parens around the `OR` operation, but forgot to close them.

Also, your statement selects only the `eye_color` attribute of each row. It's supposed to select all attributes of each matching row.

4/5

File: assessment.sql [hunk 5]
```diff
@@ -5,24 +5,42 @@ CREATE DATABASE "sql-crud-assessment";
...
 
 -- Question 5. Make each of Marilynn Escobar and Chris Whaley one inch shorter. Look them up by their names.
 
--- your code here
+UPDATE licensees
+SET height = height - 1
+WHERE (given_name = 'Marilynn' AND surname = 'Escobar') AND (given_name = 'Chris' AND surname = 'Whaley');
```
Comment (srhoulam) [inline assessment.sql:40]: This statement updates only rows where the licensee's name is both Marilynn Escobar and Chris Whaley at the same time, i.e., no rows. Here is the WHERE clause you want:

``` sql
WHERE (given_name = 'Marilynn' AND surname = 'Escobar') OR (given_name = 'Chris' AND surname = 'Whaley');
```

4/5

File: assessment.sql [hunk 6]
```diff
@@ -5,24 +5,42 @@ CREATE DATABASE "sql-crud-assessment";
...
 
 -- Question 6. Remove Dylan Rich and Teresita Myers from the table of licensees.
 
--- your code here
+DELETE FROM licensees
+WHERE (given_name = 'Dylan' AND surname = 'Rich') AND (given_name = 'Teresita' AND surname = 'Myers')
+;
```
Comment (srhoulam) [inline assessment.sql:46]: Same mistake as above. 4/5

**Comment by srhoulam** [comment]
5.4/6

**Pull request closed without merging**